a better list of teas

easement opened the pull request:
I parsed a tea catalog recently for a project and noticed it had a wider selection of items and it could be useful for this project.

hugovk commented on data/foods/tea.json:
```diff
@@ -1,182 +1,369 @@
+     "Lumbini Estate FBOPF Ex. Sp.",
+     "Lumbini Estate FBOPF Ex. Spl.",
+     "Lung Jing Te Ji",
+     "Lung-Ching:",
```
Remove trailing  `:`

hugovk commented on data/foods/tea.json:
```diff
@@ -1,182 +1,369 @@
+     "Tea Bank FBOPF Ex. Sp.",
+     "Thurbo Estate FTGFOP1 Moonlight",
+     "Tie-Guan-Yin",
+     "Tie-Guan-Yin:",
```
Remove this line. It duplicates the previous line and adds a `:`.

hugovk commented on data/foods/tea.json:
```diff
@@ -1,182 +1,369 @@
+     "Rooibos Vanilla",
+     "Rose Congou",
+     "Rose Hips",
+     "Rote Grütze (Red Groats)",
```
The following line is "Rote Grütze", so maybe change this to "Red Groats"?

easement commented:
@hugovk

1. added back in the original set
2. title cased them.
3. alphabetized them.
4. took out the redundant `Tea` in the names
5. cleaned out the `:`s

hugovk commented on data/foods/tea.json:
```diff
@@ -1,27 +1,46 @@
 {
-  "description": "types of tea",
+  "description": "types of",
```
Maybe this one line can keep the " tea" suffix :)

easement replied:
sloppy find/replace on my part. good catch.

easement replied:
done.

easement commented:
it looked like it was getting almost there, so I fixed the description and did a rebase to keep the history clean.

easement commented:
Anything else needed?
